src/Tizen.WebView/Tizen.WebView/Context.cs

```diff
@@ -38,6 +39,452 @@ public enum CacheModel
+        /// </summary>
+        MediaPlayPause,
+        /// <summary>
+        /// Media mute.
+        /// </summary>
+        MediaMute,
+        /// <summary>
+        /// Dictation alternative.
+        /// </summary>
+        DictationAlternative,
+        /// <summary>
+        /// Select all.
+        /// </summary>
+        SelectAll,
+        /// <summary>
+        /// Select word.
+        /// </summary>
+        SelectWord,
+        /// <summary>
+        /// Text selection mode.
+        /// </summary>
+        TextSelectonMode,
+        /// <summary>
+        /// Clipboard.
+        /// </summary>
+        Clipboard,
+        /// <summary>
+        /// Drag.
+        /// </summary>
+        Drag,
+        /// <summary>
+        /// Translate.
+        /// </summary>
+        Translate,
+        /// <summary>
+        /// Copy link data.
+        /// </summary>
+        CopyLinkData,
+        /// <summary>
+        /// If app want to add customized item, use enum value after #EWK_CONTEXT_MENU_ITEM_BASE_APPLICATION_TAG.
+        /// </summary>
+        BaseApplicationTag = 10000
+    }
+
+    /// <summary>
+    /// This class provides the properties of Context Menu item.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItem
+    {
+        private IntPtr _menuItemHandle;
+
+        internal ContextMenuItem(IntPtr handle)
+        {
+            _menuItemHandle = handle;
+        }
+
+        /// <summary>
+        /// Gets Tag of the context menu item.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItemTag GetItemTag()
+        {
+            return (ContextMenuItemTag) Interop.ChromiumEwk.ewk_context_menu_item_tag_get(_menuItemHandle);
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Context Menu.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenu
+    {
+        private IntPtr _menuHandle;
+
+        internal ContextMenu(IntPtr handle)
+        {
+            _menuHandle = handle;
+        }
+
+        /// <summary>
+        /// Gets the context menu items count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int MenuItemCount()
+        {
+            return Interop.ChromiumEwk.ewk_context_menu_item_count(_menuHandle);
+        }
+
+        /// <summary>
+        /// Gets Nth item of the context menu.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItem GetNthItem(int n)
+        {
+            IntPtr ItemPtr = Interop.ChromiumEwk.ewk_context_menu_nth_item_get(_menuHandle, n);
+            ContextMenuItem item = new ContextMenuItem(ItemPtr);
+            return item;
+        }
+
+        /// <summary>
+        /// Removes item from the context menu.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool RemoveItem(IntPtr item)
```

Comment:
|_menuItemHandle| is private member, how can developer get it?

Reply:
From getItem above function